src/services/query.ts

```diff
@@ -95,6 +95,9 @@ export function renderLogQLFieldFilters(filters: AdHocVariableFilter[]) {
 }
 
 function renderFilter(filter: AdHocVariableFilter) {
+  if (filter.value === EMPTY_VARIABLE_VALUE) {
```

Comment:
Silly, but took me a minute to understand this. Maybe adding a comment would be nice.

Reply:
Done!